allure-report-plugin-api/src/main/java/ru/yandex/qatools/allure/plugins/ReportConfig.java

```diff
@@ -25,4 +25,12 @@
     @DefaultValue("%s")
     @Property("allure.tests.management.pattern")
     String getTmsPattern();
+
+    /**
+     * Pattern containing stories tracker base URL and one %s placeholder which will be replaced by story id.
+     * Example: http://example.com/%s and @Stories(value = "story name" id = "SOME-123") will give you http://example.com/SOME-123
+     */
+    @DefaultValue("%s")
```

Comment:
@ctapobep You proposal look's cleaner than my current implementation (one id for all stories). I may change it, but the question is if additional annotation, and very similar to @`Stories`, would not confuse the end users. Currently I don't see an easy way to track story name and id, this would require a lot of changes, but since we need story id on Test case details page, I guess we may leave the relation for now.
@baev what do you think about @ctapobep proposal?